File: Python/password.py
"""Implement a program that generates a random password using a specific set of characters"""
import secrets, string

"""Build a set of characters that may be used in a password"""
letters = string.ascii_letters
digits = string.digits
special_chars = string.punctuation

# Store the set in a corresponding variable
alphabet = letters + digits + special_chars

# Determine how long a password should be
pwd_length = 12

# Generate a password
pwd = ''
for _ in range(pwd_length):
  pwd += ''.join(secrets.choice(alphabet))

# Print out the password
print(pwd)

File: JavaScript/password.js
/* Implement a fucntion that generates a one time password 
    comprised of digits, which is six characters long */ 
    function generateOTP() {
    var digits = '0123456789';
    let OTP = ' ';
    for (let i = 0; i < 6; i++) {
        OTP += digits[Math.floor(Math.random() * 10)];
    }
    console.log(OTP);
}

generateOTP()